docs/source/resources/transition_to_ft_v1.0.ipynb

```diff
@@ -0,0 +1,801 @@
+    "                  target_dataframe_name='items',\n",
+    "                  features_only=True)\n",
+    "features"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3154734d",
+   "metadata": {},
+   "source": [
+    "In addition the `dfs` argument `ignore_entities` was renamed to `ignore_dataframes` and `ignore_variables` was renamed to `ignore_columns`. Similarly if specifying primitive options all references to `entities` should be replaced with `dataframes` and references to `variables` should be replaced with columns. For example, the primitive option of `include_groupby_entities` is now `include_groupby_dataframes` and `include_variables` is now `include_columns`.\n",
+    "\n",
+    "The basic call to `featuretools.calculate_feature_matrix` remains unchanged if passing in an EntitySet along with a list of features to caluculate. However, users calling `calculate_feature_matrix` by passing in a list of `entities` and `relationships` should note that the `entities` argument has been renamed to `dataframes` and the values in the dictionary values should now include Woodwork logical types instead of Featuretools `Variable` classes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "456da22e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_matrix = ft.calculate_feature_matrix(features=features, entityset=es)\n",
+    "feature_matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b87489cf",
+   "metadata": {},
+   "source": [
+    "In addition to the changes in argument names, there are a couple other changes to the returned feature matrix that users should be aware of. First, because of slight differences in the way Woodwork defines column types compared to how the prior Featuretools implementation did, there can be some differences in the features that are generated between old and new versions. The most notable impact is in the way foreign key columns are handled. Previously, Featuretools treats all foreign key (previously `Id`) columns as categorical columns, and would generate appropriate features from these columns. Starting in version 1.0, foreign key columns are not constrained to be categorical, and if they are another type such as `Integer`, features will not be generated from these columns. Manually converting foreign key columns to `Categorical` as shown above will result in features much closer to those achieved with previous versions.\n",
+    "\n",
+    "Also, because Woodwork's type inference process differs from the previous Featuretools type inference process an EntitySet may have column types identified differently. This difference in column types could impact the features that are generated. If it is important to have the same set of features, check all of the logical types in the EntitySet dataframes and update them to the expected types if there are columns that have been inferred as unexpected types.\n",
```

Comment:
grammar nitpick - add a comma after process: `Also, because Woodwork's type inference process differs from the previous Featuretools type inference process, an EntitySet may have column types identified differently`